gems/pending/VixDiskLib/VixDiskLib.rb

```diff
@@ -142,14 +138,10 @@ def self.start_service
   def self.get_uri(reader)
     if reader.eof
       #
-      # Error - unable to read the port number written into the pipe by the child (Server).
+      # Error - unable to read the URI with port number written into the pipe by the child (Server).
       #
       raise VixDiskLibError, "ERROR: VixDiskLib.connect() Unable to determine port used by VixDiskLib Server."
     end
-    uri_selected = reader.gets.split("URI:")
-    if uri_selected.length != 2
-      raise VixDiskLibError, "ERROR: VixDiskLib.connect() Unable to determine port used by VixDiskLib Server."
-    end
-    uri_selected[1].chomp
+    reader.gets
```

Comment:
Using `gets` then relying on the fact it doesn't have a trailing newline seems confusing; use `read` to be clear that there's nothing left when we're done?